pkgs/desktops/gnome-3/apps/gnome-network-displays/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv
+, fetchurl
+# native
+, meson
+, ninja
+, cmake
+, pkg-config
+, gettext
+, desktop-file-utils
+, appstream-glib
+, wrapGAppsHook
+, python3
+# Not native
+, gst_all_1
+, gsettings-desktop-schemas
+, gtk3
+, glib
+, networkmanager
+, libpulseaudio
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-network-displays";
+  version = "0.90.4";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "04snnfz5jxxpjkrwa7dchc2h4shszi8mq9g3ihvsvipgzjw3d498";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    cmake
```

Comment:
Are you sure `cmake` is needed? Meson supports it for finding dependencies but typically, `pkg-config` will be sufficient.

Reply:
Indeed, somehow it gave me an error earlier saying it didn't find it.

Reply:
See https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/meson-cmake.md

Reply:
LOL. I wish I could give two stars to that repo :).